src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx

```diff
@@ -122,11 +129,30 @@ function ConfirmationStep({policyID, backTo}: ConfirmationStepProps) {
                         isAlertVisible={!!errorMessage}
                         isDisabled={isOffline}
                         isLoading={issueNewCard?.isLoading}
-                        onSubmit={submit}
+                        onSubmit={() => setIsValidateCodeActionModalVisible(true)}
                         buttonText={translate('workspace.card.issueCard')}
                     />
                 </View>
             </ScrollView>
+            <ValidateCodeActionModal
+                handleSubmitForm={submit}
+                isLoading={issueNewCard?.isLoading}
+                sendValidateCode={() => User.requestValidateCodeAction()}
+                validateError={validateLoginError}
+                clearError={() => {
+                    User.clearContactMethodErrors(addSMSDomainIfPhoneNumber(pendingContactAction?.contactMethod ?? contactMethod), 'addedLogin');
+                }}
+                onClose={() => {
+                    if (validateLoginError && pendingContactAction?.contactMethod) {
+                        User.clearContactMethod(pendingContactAction?.contactMethod);
```

Comment:
Same here, why do we need `User.clearContactMethod(pendingContactAction?.contactMethod);` here?